src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/ConstructorParametersShouldMatchPropertyAndFieldNamesTests.Fixer.cs

```diff
@@ -0,0 +1,384 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Testing;
+using Xunit;
+using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
+    Microsoft.NetCore.Analyzers.Runtime.ConstructorParametersShouldMatchPropertyAndFieldNamesAnalyzer,
+    Microsoft.NetCore.Analyzers.Runtime.ConstructorParametersShouldMatchPropertyAndFieldNamesFixer>;
+using VerifyVB = Test.Utilities.VisualBasicCodeFixVerifier<
+    Microsoft.NetCore.Analyzers.Runtime.ConstructorParametersShouldMatchPropertyAndFieldNamesAnalyzer,
+    Microsoft.NetCore.Analyzers.Runtime.ConstructorParametersShouldMatchPropertyAndFieldNamesFixer>;
+
+namespace Microsoft.NetCore.Analyzers.Runtime.UnitTests
+{
+    public class ConstructorParametersShouldMatchPropertyAndFieldNamesFixerTests
+    {
+        [Fact]
+        public async Task CA1071_ClassSinglePropDoesNotMatch_CSharp()
+        {
+            await VerifyCSharpCodeFixAsync(@"
+                using System.Text.Json.Serialization;
+
+                public class C1
+                {
+                    public int FirstProp { get; }
+
+                    public object SecondProp { get; }
+
+                    [JsonConstructor]
+                    public C1(int [|firstDrop|], object secondProp)
+                    {
+                        this.FirstProp = firstDrop;
+                        this.SecondProp = secondProp;
+                    }
+                }",
```

Comment:
Same for all strings.

```suggestion
using System.Text.Json.Serialization;

public class C1
{
    public int FirstProp { get; }

    public object SecondProp { get; }

    [JsonConstructor]
    public C1(int [|firstDrop|], object secondProp)
    {
        this.FirstProp = firstDrop;
        this.SecondProp = secondProp;
    }
}",
```